app/views/layouts/exp_atom/_editor.html.haml

```diff
@@ -23,25 +23,24 @@
     %fieldset
       .toolbar-pf-actions
         .form-group
-          %button
-            - t = _('Commit expression element changes')
-            = link_to(image_tag(image_path('toolbars/commit.png'), :alt => t),
-              {:action => 'exp_button', :pressed => 'commit'},
-              "data-miq_sparkle_on"  => true,
-              "data-miq_sparkle_off" => true,
-              :remote                => true,
-              "data-method"          => :post,
-              :title                 => t)
-          %button
-            - t = _("Discard expression element changes")
-            = link_to(image_tag(image_path('toolbars/discard.png'), :alt => t),
-              {:action => 'exp_button', :pressed  => 'discard'},
-              "data-miq_sparkle_on"  => true,
-              "data-miq_sparkle_off" => true,
-              :remote                => true,
-              "data-method"          => :post,
-              :title                 => t)
-
+          - t = _('Commit expression element changes')
```

Comment:
@epwinchell could you please test it and give me an exact and working example? 
Because I probably don't know how you mean it.
I mean, wit the code around, not just with "..."
It just does not work as simply as you mentioned. I tried it in many ways, without success. Some of my colleagues tried it, too, and also without successs.
Originally, the problem was `button` element.
I know that styling is important, but as I think, more important is if the button works properly.
So I removed the button element and it worked perfectly. Adding some classes was not a problem, it was possible to add them very easily to my original solution.I know that you would like to see nicer solution but my solution worked comparing to the "styling we need". This all makes me to ask: is really button element required to use there? Why? We survived to these days without it.. If yes, how would you put there all the options to make it work properly?